core/i18n.py

```
@@ -146,16 +138,16 @@ def get_string_with_fallback(
     ) -> str:
         node = self.data.query_node(key)
         if node:
-            return node.value  # 1. 如果本地化字符串存在，直接返回
+            return node.value
+            # 1. 如果本地化字符串存在，直接返回
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)